test/AutoDiff/derived_differentiable_properties.swift

```diff
@@ -7,7 +7,7 @@ public struct Foo : Differentiable {
 }
 
 // CHECK-AST-LABEL: public struct Foo : Differentiable {
-// CHECK-AST:   @sil_stored @differentiable()
+// CHECK-AST:   @sil_stored @differentiable(wrt: (self))
```

Comment:
I love that things have a canonical printout now when you use checked parameter indices.